README.md

```diff
@@ -86,6 +87,7 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
 - [Payloads All The Things](https://swisskyrepo.github.io/PayloadsAllTheThingsWeb/) - A list of useful payloads and bypasses for Web Application Security
 - [XSS Payloads](http://www.xss-payloads.com/) - The wonderland of JavaScript unexpected usages, and more
 - [exploitalert.com](https://www.exploitalert.com/search-results.html) - Database of Exploits
+- [Reverse Shell generator](https://www.revshells.com/) - Plug in attacker machine IP and port and generate any kind of reverse shell on the fly.
```

Comment:
This should be`Online Reverse Shell generator with Local Storage functionality, URI & Base64 Encoding, MSFVenom Generator, and Raw Mode.`